packages/create-vue-termui/template-ts/package.json

```diff
@@ -23,7 +23,6 @@
     "typescript": "^4.5.4",
     "unplugin-auto-import": "^0.6.1",
     "unplugin-vue-components": "^0.18.0",
-    "vite": "^2.5.10",
```

Comment:
I don't think this should be removed

Reply:
It seems a devDependencies, we can use it from root deps, any special reason ?